src/game/game_init.c

```diff
@@ -66,6 +66,8 @@ u8 gIsConsole;
 #ifdef WIDE
 u8 gWidescreen;
 #endif
+u8 *gAreaSkyboxStart[7];
```

Comment:
is there an area count define you can use to size these instead of hardcoding to 7?

Reply:
same thing in the header